releasenotes/notes/parse-snmp-configuration-1aaeefbb06d07380.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+
+features:
+  - |
+    parse the snmp configuration from the agent and pass it to the integrated snmpwalk command in case the customer only provides an ip address
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** 
‏
```suggestion
    [snmp] Parse the snmp configuration from the agent and pass it to the integrated snmpwalk command in case the customer only provides an ip address.
```